build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt

```diff
@@ -14,7 +14,7 @@ import org.gradle.kotlin.dsl.withType
 
 private const val TASK_GROUP = "Verification"
 internal const val OUTPUT_PATH = "reports/metrics/"
-internal const val S3_ARTIFACT_SIZE_METRICS_BUCKET = "" // TODO: FILL IN BUCKET
+internal const val S3_ARTIFACT_SIZE_METRICS_BUCKET = "artifact-size-metrics-aws-sdk-kotlin"
```

Comment:
question/naming: will this bucket will also contain the metrics for smithy-kotlin (and possibly aws-crt-kotlin)? Maybe the name can be made more generic to reflect that

Reply:
Do we need to specify the team name if it's always going to be in our team account? Why not just `artifact-size-metrics`?

Reply:
I thought the bucket name would've been taken already. But this works better. Good call :+1: